library/ajax/template_context_search.php

```diff
@@ -37,16 +37,15 @@
     cl2.cl_deleted = 0 And
     cl3.cl_list_type = 3 And
     cl3.cl_deleted = 0 And
-    cl2.cl_list_item_long Like ? And
-    cl3.cl_creator = ?
+    cl2.cl_list_item_long Like ?
 Group By
     cl2.cl_list_item_long
 createQuery;
 
 $search = $_GET['search'];
 $eSearch = "%" . $search . "%";
 $results = [];
-$r = sqlStatementNoLog($cq, array($eSearch, (int)$_SESSION['authUserID']));
+$r = sqlStatementNoLog($cq, array($eSearch));
```

Comment:
There's a reason the query includes the creator id, do you know what it is before removing to make this work?
Are you sure that now all user templates aren't exposed?
Does this affect test templates. I doubt it but worth testing to be sure.

I don't want to have to did into this code as last time I worked in it it was working and my clinic uses both text and NN extensively so I don't want to see any more issues I have to get calls on:)
So far they have not had any issues but only had running 702 for 5 business days.

Reply:
@sjpadgett I don't have enough testing to answer your questions. I searched the code trying to find why the authUserID was there. I went all the way back to v5. The auth user ID is in v5 but I am too ignorant to see the purpose. In my use case, there is only one person creating the templates. I can tell you that when removed the parameter. The shared template showed to the person that wanted to use it. They don't show up to everyone and only the ones that are selected show to the one that requested the share. I would have to do more digging and trapping to figure out more than that.